scripts/tracing/parse_ctf.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Script to parse CTF data and print to the screen in a custom and colorful
```

Comment:
Do I understand the role of this script? This script looks specific to babeltrace. Does it use babeltrace2 to interpret the CTF channel and metadata, and this script works to customize the display using babeltrace APIs?

Reply:
this is just a helper script to read the generated data on the command line.